src/extensions/default/bramble/lib/Theme.js

```diff
@@ -0,0 +1,73 @@
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4,
+maxerr: 50, browser: true */
+/*global define, brackets */
+
+/**
+ * This extension provides in-editor livepreview through an iframe,
+ * and leverages the experimental Multi Browser implementation of brackets
+ * (see https://github.com/adobe/brackets/tree/master/src/LiveDevelopment/MultiBrowserImpl)
+ */
+define(function (require, exports, module) {
+    "use strict";
+
+    var ThemeManager     = brackets.getModule("view/ThemeManager");
+    var ThemePreferences = brackets
+                            .getModule("preferences/PreferencesManager")
+                            .getExtensionPrefs("themes");
+    var Path             = brackets.getModule("filesystem/impls/filer/BracketsFiler").Path;
+    var basePath = PathUtils.directory(window.location.href);
+    var themePath = Path.join(basePath, 'extensions/default');
+
+    // Store the current theme, and load defaults as if
+    // they were third party themes
+    var currentTheme = 'light-theme';
+
+    function toggle(data) {
+        var newTheme;
+
+        if (data.theme) {
+            currentTheme = newTheme = data.theme === 'light-theme' ? "dark-theme" : "light-theme";
+        } else {
+            currentTheme = newTheme = currentTheme === 'light-theme' ? "dark-theme" : "light-theme";
+        }
+
+        ThemePreferences.set("theme", newTheme);
```

Comment:
You're doing very similar things in this function to `setTheme`, so I'd recommend you call it instead.